src/scenes/home/opCodeCon/opCodeCon.css

```diff
@@ -0,0 +1,12 @@
+.hero {
+  background-image: url('./raleigh.jpg');
+  text-align: left;
```

Comment:
The CSS rule `text-align: left` can likely be removed, it doesn't appear to be doing anything.